src/sentry/tasks/store.py

```diff
@@ -563,10 +567,19 @@ def _do_save_event(
             if not isinstance(data, dict):
                 data = dict(data.items())
             processing.event_processing_store.store(data)
+
+            event_type = data.get("type")
+            if event_type == "transaction" and options.get("rc-processing-split-write-double"):
```

Comment:
With something as sensitive as rc-processing, you should considering using a rate based option that will allow you to sample traffic in for the dual writes. Once we've verified the cluster works correctly, the rate can up or to 1.0.

Reply:
Thinking about this more, you'd need to have consistent rollout rates. We've done this in the past a few times. Most recently in #12909

Reply:
Thought about this more, and I don't think you'd need the key based selector for dual-write as you can go from 0.01 -> 1.0 which gives you control over how fast the new redis will be populated.

The delete operation should probably run if the option is > 0. to avoid data being stranded in transaction processing.